scipy/optimize/_linprog.py

```diff
@@ -187,18 +187,22 @@ def linprog(c, A_ub=None, b_ub=None, A_eq=None, b_eq=None,
 
     Alternatively, that's:
 
-    minimize::
+        - minimize ::
 
-        c @ x
+            c @ x
 
-    such that::
+        - such that ::
 
-        A_ub @ x <= b_ub
-        A_eq @ x == b_eq
-        lb <= x <= ub
+            A_ub @ x <= b_ub
+            A_eq @ x == b_eq
+            lb <= x <= ub
 
     Note that by default ``lb = 0`` and ``ub = None`` unless specified with
```

Comment:
Was "unless specified by bounds" was not enough to communicate that the bounds did not have to be positive?

I don't think it should be "unless specified with `bounds`" followed by "however", though, because this new statement agrees with "unless specified by `bounds`".

Reply:
It was not clear enough when I first started to use `linprog`.
What do you propose? `"Furthermore, note that ..."`?

Reply:
Personally, I'd propose that the user look at the documentation for ``bounds``. Perhaps that could be more explicit that both negative and positive values are allowed. This is just the introduction to the function and I'm not sure that much detail belongs here.

If you think many people think that this is a common problem (e.g. if you've talked to someone else and they've had the same misunderstanding, you could add a `.. note::`` after bounds.

```
.. note::

    There is no need to manually formulate the linear
    programming problem in terms of non-negative variables; 
    lower and upper elements of ``bounds`` can be set to 
    any real or ``None``.
```

A few other thoughts:

"often termed slack variables" - in my experience, this term only refers to variables that are added to the problem to handle inequality constraints. This is not all positive variables.

"``min`` in ``bounds``" - "min" is not defined until you get to `bounds`

"negative or real valued ``x``. " - negative is a subset of reals, so would "any real" do?

Reply:
I put the comments under the bounds parameter in https://github.com/scipy/scipy/pull/17867/commits/0759eb7d2dbf9024ce364283025cc6a7055e6c79.
If that does still not go in the right direction, I'll close this PR to save review time for more important PRs.